src/ray/core_worker/common.h

```diff
@@ -97,6 +95,48 @@ enum class StoreProviderType { PLASMA };
 
 enum class TaskTransportType { RAYLET };
 
+/// Translate from ray::rpc::Language to Language type (required by raylet client).
+///
+/// \param[in] language Language for a task.
+/// \return Translated task language.
+inline ::Language ToTaskLanguage(ray::rpc::Language language) {
```

Comment:
Having the same name for both of these functions might make it harder to read. For clarity, it would be helpful to specify the "direction" of the translation in the function name, like:
`::Language ToCoreTaskLanguage(ray::rpc::Language language)`
and for the one below:
`ray::rpc::Language ToRPCTaskLanguage(::Language language)`

Reply:
Done.